addons/explore-editor-theme/tabs/EditorIcons.tscn

```diff
@@ -102,24 +106,30 @@ rect_min_size = Vector2( 0, 12 )
 [node name="IconName" type="Label" parent="Layout/IconView/IconPanel"]
 margin_top = 96.0
 margin_right = 280.0
-margin_bottom = 110.0
-custom_fonts/font = SubResource( 5 )
+margin_bottom = 127.0
+custom_fonts/font = SubResource( 3 )
 
 [node name="Padding2" type="Control" parent="Layout/IconView/IconPanel"]
-margin_top = 114.0
+margin_top = 131.0
 margin_right = 280.0
-margin_bottom = 126.0
+margin_bottom = 143.0
 rect_min_size = Vector2( 0, 12 )
 
 [node name="IconCode" parent="Layout/IconView/IconPanel" instance=ExtResource( 2 )]
 anchor_right = 0.0
 anchor_bottom = 0.0
-margin_top = 130.0
+margin_top = 147.0
 margin_right = 280.0
-margin_bottom = 230.0
+margin_bottom = 266.0
+
+[node name="IconSaver" parent="Layout/IconView/IconPanel" instance=ExtResource( 6 )]
+margin_top = 270.0
+margin_bottom = 309.0
 
 [node name="EmptyPanel" type="Control" parent="Layout/IconView"]
-margin_left = 992.0
-margin_right = 1272.0
-margin_bottom = 656.0
+margin_left = 736.0
+margin_right = 1016.0
+margin_bottom = 536.0
 rect_min_size = Vector2( 280, 0 )
+
+[connection signal="filesystem_changed" from="Layout/IconView/IconPanel/IconSaver" to="." method="_notify_filesystem_change"]
```

Comment:
Same for signal connections (for whatever reason I forgot to mark this one as well).